### PR TITLE
Add nodejs websocket service

### PR DESCRIPTION
**Pull request by Thushani-Jayasekera**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by chameerar — CHANGES_REQUESTED** [review]

File: .samples/websocket-chat-app.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+displayName: WebSocket Chat Application
+description: Provides simultaneous two-way communication over a single TCP connection.
+componentType: service
+buildPack: node
```
[the remark this answers is not in the document]
Comment (Thushani-Jayasekera) [reply .samples/websocket-chat-app.yaml:4]: I checked with that as well previously. still the build failed.

File: .samples/websocket-chat-app.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+repositoryUrl: https://github.com/wso2/choreo-samples/
+componentPath: /websocket-chat-app/websocket-chat-service-nodejs
```
Comment (chameerar) [inline .samples/websocket-chat-app.yaml:6]: Let's use `/websocket-chat-app` as the componentPath here.  Example - .samples/reading-list-app.yaml

Also let's add a readme to /websocket-chat-app to describe the contents of the subfolders. Example - reading-list-app/README.md

**Review by chameerar — CHANGES_REQUESTED** [review]

File: .samples/websocket-chat-app.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (chameerar) [reply .samples/websocket-chat-app.yaml:4]: Looks like the issue is with the given componentPath. That can be fixed as described in https://github.com/wso2/choreo-samples/pull/159#discussion_r1803080655.

**Review by chameerar — APPROVED** [review]
LGTM

**Pull request merged**